data/strings/types_strings.txt

```diff
@@ -27084,3 +27082,84 @@
     vi = Địa điểm tổ chức sự kiện
     zh-Hans = 活动场所
     zh-Hant = 活動場所
+
+  [type.shop.auction]
+    en = Auction
+    ar = مزاد علني
+    be = Аўкцыён
+    bg = Търг
+    ca = Subhasta
+    cs = Aukce
+    da = Auktion
+    de = Auktion
+    el = Δημοπρασία
+    es = Subasta
+    et = Oksjon
+    eu = Enkantea
+    fa = حراج
+    fi = Huutokauppa
+    fr = Vente aux enchères
+    he = מכירה פומבית
+    hu = Aukció
+    id = Lelang
+    it = Asta
+    ja = オークション
+    ko = 경매
+    mr = लिलाव
+    nb = Auksjon
+    nl = Veiling
+    pl = Aukcja
+    pt = Leilão
+    ro = Licitație
+    ru = Аукцион
+    sk = Aukcia
+    sv = Auktion
+    sw = Mnada
+    th = ประมูล
+    tr = Müzayede
+    uk = Аукціон
+    vi = Bán đấu giá
+    zh-Hans = 拍卖会
+    zh-Hant = 拍賣
+
+  [type.shop.collector]
```

Comment:
Here you can keep "Collectables" or any better explanation term unlike in categories.